rendermodules/em_montage_qc/schemas.py

```diff
@@ -54,7 +54,8 @@ def add_match_collection_owner(self, data):
 
 
 class DetectMontageDefectsParametersOutput(argschema.schemas.DefaultSchema):
-    output_html = Str(
+    output_html = argschema.fields.List(
```

Comment:
Does `output_html` still work?  I don't see where this is used as a list.

Reply:
output_html is used by blue_sky to link the plot to the web interface showing status of each montage set

Reply:
this would be a list if minZ and maxZ is a range and not hold the same value.

Reply:
I misread this -- Thought it was in the input schema and now it makes a lot more sense.